client/src/main/com/sinch/sdk/domains/verification/adapters/VerificationService.java

```diff
@@ -22,28 +24,33 @@ public class VerificationService implements com.sinch.sdk.domains.verification.V
   private final HttpClient httpClient;
   private VerificationsService verifications;
   private StatusService status;
-  private final Map<String, AuthManager> authManagers;
+  private Map<String, AuthManager> authManagers;
 
   public VerificationService(Configuration configuration, HttpClient httpClient) {
     this.configuration = configuration;
     this.httpClient = httpClient;
 
+    // by default, unified id/secret from configuration are used but can be super sed if
+    // "useSecrets"
+    // is called after initialization
+    useSecrets(
+        configuration.getKeyId(),
+        Base64.getEncoder()
```

Comment:
I'm not sure we need to encode the secret as it is already provided as encoded in the applications dashboard. I don't think a customer will base64-decode it before using it in the application.

Reply:
This code branch is for using the unified sinch credentials and in they are not base64 encoded.

Currently it do not work on runtime because server API won't accept it we need to call the dedicated useSecrets function to set proper application credentials; but if they was accepted by server, it could be the expected workflow